### PR TITLE
Display Identity Claims in Attribute and Application Attribute Sections

### PR DESCRIPTION
**Pull request by DilshanSenarath**
### Related Issues
- https://github.com/wso2/product-is/issues/19157

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Comment by DilshanSenarath** [comment]
https://github.com/wso2/identity-apps/assets/74205483/f8430ad8-1d27-4ac9-ad94-7bfdb9d65650

File: apps/console/src/extensions/components/user-stores/components/edit/userstore-attribute-mappings.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com).
```
Comment (brionmario) [inline apps/console/src/extensions/components/user-stores/components/edit/userstore-attribute-mappings.tsx:2]:
```suggestion
 * Copyright (c) 2022-2024, WSO2 LLC. (https://www.wso2.com).
```

Comment (brionmario) [reply apps/console/src/extensions/components/user-stores/components/edit/userstore-attribute-mappings.tsx:2]: Lets fix other places as well.

File: apps/console/src/extensions/configs/application.tsx [hunk 2]
```diff
@@ -89,6 +89,12 @@ const featureConfig: FeatureConfigInterface = window[ "AppUtils" ].getConfig().u
  * @returns boolean
  */
 const isIdentityClaim = (claim: ExtendedClaimInterface | ExtendedExternalClaimInterface): boolean => {
+    const excludeIdentityClaims: boolean = window[ "AppUtils" ]?.getConfig()?.ui?.excludeIdentityClaims;
```
Comment (brionmario) [inline apps/console/src/extensions/configs/application.tsx:92]: Can't we pass this as an argument, so we can use the `useSelector` hook from component level and avoid accessing from window?

File: apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2 [hunk 3]
```diff
@@ -1699,6 +1699,7 @@
         {% endif %}
         "isSignatureValidationCertificateAliasEnabled": {{ console.applications.ui.certificate_alias_enabled }},
         "listAllAttributeDialects": {{ console.list_all_attribute_dialects }},
+        "excludeIdentityClaims": {{ console.exclude_identity_claims }},
```
Comment (brionmario) [inline apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2:1702]: Add a null check for the config as well.

File: apps/console/src/features/core/configs/app.ts [hunk 4]
```diff
@@ -278,6 +278,7 @@ export class Config {
                 defaultWhiteLogoUrl: window[ "AppUtils" ]?.getConfig()?.ui?.emailTemplates?.defaultWhiteLogoUrl
             },
             enableCustomEmailTemplates: window[ "AppUtils" ]?.getConfig()?.ui?.enableCustomEmailTemplates,
+            excludeIdentityClaims: window[ "AppUtils" ]?.getConfig()?.ui?.excludeIdentityClaims,
```
Comment (brionmario) [inline apps/console/src/features/core/configs/app.ts:281]: If config is undefined, set the value to true by default.

File: apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2 (hunk 3 above)
Comment (brionmario) [reply apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2:1702]: Else, there'll be a JSON parse error if the framework and console versions don't sync.

File: apps/console/src/extensions/components/user-stores/components/edit/userstore-attribute-mappings.tsx (hunk 1 above)
Comment (DilshanSenarath) [reply apps/console/src/extensions/components/user-stores/components/edit/userstore-attribute-mappings.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/5350/commits/c63c7807e17debc2bc358ac2201c6ca8adbfb074

File: apps/console/src/extensions/configs/application.tsx (hunk 2 above)
Comment (DilshanSenarath) [reply apps/console/src/extensions/configs/application.tsx:92]: `IsIdentityClaim` is only used inside the config file not in components. So, if we want to pass this using `useSelector` hook we need to pass it as chain from function to function. However, there is a one method it is not used anywhere, therefore I removed it https://github.com/wso2/identity-apps/pull/5350/commits/4cf97ff09386b2ca9eae4cea74b6131cba862bd5.
`IsIdentityClaim` is used only within the configuration file, not in components. Therefore, when passing it using the `useSelector` hook, it requires traversing through a chain of functions. However, there is one method that is not utilized anywhere; consequently, it has been removed. https://github.com/wso2/identity-apps/pull/5350/commits/4cf97ff09386b2ca9eae4cea74b6131cba862bd5

File: apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2 (hunk 3 above)
Comment (DilshanSenarath) [reply apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2]: Fixed with https://github.com/wso2/identity-apps/pull/5350/commits/7d3ce4d4b2ae15058599ef21dfd976f4b8f54974

File: apps/console/src/features/core/configs/app.ts (hunk 4 above)
Comment (DilshanSenarath) [reply apps/console/src/features/core/configs/app.ts]: Fixed with https://github.com/wso2/identity-apps/pull/5350/commits/fa3eaa3bd7ef78fc42454367b8cd8d7796438f66

**Review by brionmario — APPROVED** [review]

**Pull request merged**